tests/lizmap-ctl

```diff
@@ -130,7 +130,7 @@ case $COMMAND in
        docker exec -it --user postgres -e PGPASSWORD="lizmap1234!" lizmap${LZMBRANCH}_test_pgsql /bin/bash  -l
        ;;
     dump-pgsql)
-       docker exec -i -e PGPASSWORD="lizmap1234!" lizmap${LZMBRANCH}_test_pgsql pg_dump -U lizmap --no-owner --no-acl -n tests_projects -f /srv/lzm/tests/qgis-projects/tests/tests_dataset.sql
+       docker exec -i -e PGPASSWORD="lizmap1234!" lizmap${LZMBRANCH}_test_pgsql pg_dump -U lizmap --no-owner --no-acl --restrict-key=lizmap_test -n tests_projects -f /srv/lzm/tests/qgis-projects/tests/tests_dataset.sql
```

Comment:
Keys may contain only alphanumeric characters.

In this case you will have: `pg_dump: error: invalid restrict key`